libs/models/schema/ui/buttons/src/lib/models-schema-ui-buttons.module.ts

```diff
@@ -0,0 +1,7 @@
+import { NgModule } from '@angular/core';
```

Comment:
There's no need for this file. The module has no declarations to manage

Reply:
file removed